### DISCUSSION TITLE
Seeking Mentorship for AnimalAi Project

### DISCUSSION THREAD

**Discussion by Ganibuhari** [Kinds-of-Intelligence-CFI/animal-ai#55] [General]
Hello,
I am Gani a final-year student at Universiti Brunei Darussalam, deeply interested in conducting research on AnimalAI. I am currently looking for a mentor who could provide guidance on my project. If possible, I would love the opportunity for a 1-on-1 conversation to discuss my ideas and how to move forward in this field.

Any support or advice would be greatly appreciated, and I am more than happy to work around your schedule for a potential meeting.

Thank you for your time, and I look forward to any potential collaboration!

Best regards,
Mohamed Gani 
Email: 22B6024@ubd.edu.bn

**Comment by alhasacademy96** [comment]
Hey Mohamed! Nice to meet you :)

We're so happy you're taking an interest in Animal-AI!

Regarding your request, if I understood correctly, you are seeking close, regular technical help for your research project. As it stands, we do not offer bespoke or regular technical help to users (the docs cover almost everything on how to use Animal-AI) but we can of course have a one-off meeting set up to answer your questions. This would be with myself. 

Let me know if this helps :)

Ibrahim
Lead Software Engineer @ Animal-AI

**Reply by Ganibuhari** [reply]
Hi Mr. Ibrahim,

Thank you so much for your response! I appreciate the offer for a one-off meeting and completely understand the limitations on regular support. I’ve had some delays in accessing my GitHub account due to OTP issues, which is why I'm reaching out a bit later than planned.

A Google Meet would be fantastic! Please let me know a date and time that works best for you, and I’ll make sure to be available.

Thank you again!

Best,
Mohamed Gani
22B6024@ubd.edu.bn
